lib/configuru/config_methods.rb

```diff
@@ -77,47 +78,49 @@ def param(name,options={})
     def lock(flag=true)
       @locked = flag
     end
-    def is_locked
+
+    def locked?
       @locked = false unless instance_variable_defined?(:@locked)
       @locked
     end
+
     def param_names
       self.class.param_names
     end
+
     def set_parent_object(object)
       @__parent_object = object
     end
-    
+
     def configure(options={})
-      Hash(options).each_pair do |name,value|
+      Hash(options).each_pair do |name, value|
         if name.to_sym == :options_source
           self.options_source = value
         else
-          send("#{name.to_s}=",value)
+          send("#{name}=", value)
         end
       end
       yield self if block_given?
       self
     end
+
     def options_source=(value)
       sub_options = case value
-        when Hash, Array then value
-        when IO, StringIO, Tempfile then 
-          YAML.load(value)
-        when String, Pathname
-          output = {}
-          File.open(value,"r") { |f| output = YAML.load(f) }
-          output
-        else
-          raise ArgumentError.new("Wrong argument class for options_source: #{value.class}")
-      end
+                    when Hash, Array then value
+                    when IO, StringIO, Tempfile then
+                      YAML.load(value)
+                    when String, Pathname
+                      output = {}
+                      File.open(value, 'r') { |f| output = YAML.load(f) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.